VidaToDo/Features/Todos/Todo List/Interface/TodoListTableViewController.swift

```diff
@@ -41,7 +32,13 @@ class TodoListTableViewController: UIViewController, UITableViewDelegate, Routab
     override func viewDidLoad() {
         super.viewDidLoad()
         setupView()
-        setupSubscriptions()
+        subscribeTheViewModel(viewModel)
```

Comment:
Ouh. I don't know about this name.

Reply:
In 'subscribeTheViewModel' we are subscribing the view model to the view controller's observables. While in 'subscribeToViewModel'  the view controller subscribes to the view model's observable.

Alt. naming proposal?